src/sentry/static/sentry/app/views/settings/projectAlerts/ruleRow.tsx

```diff
@@ -162,15 +162,14 @@ const RuleName = styled(Link)`
   font-weight: bold;
 `;
 
-const Conditions = styled('div')`
-  display: flex;
-  flex-direction: column;
-  justify-content: space-between;
-  height: 100%;
+const Listing = styled('div')`
+  display: grid;
+  grid-gap: ${space(1)};
 `;
 
-// For tests
-const Actions = styled('div')<HasBorderProp>`
+const Conditions = styled(Listing)``;
```

Comment:
are you sure about it? I would replace Conditions with Listing

Reply:
Updated